projects/demo/tsconfig.app.json

```diff
@@ -2,9 +2,14 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "outDir": "./out-tsc/app",
     "types": []
   },
+  "angularCompilerOptions": {
+    "enableI18nLegacyMessageIdFormat": false,
+    "strictInjectionParameters": false, // TURN BACK ON
```

Comment:
What's the purpose of these comments? When should this be turned back on?

Reply:
I don't know. It was there before.